presto-tests/src/test/java/com/facebook/presto/featuretests/TestExceptBase.java

```diff
@@ -0,0 +1,95 @@
+/*
```

Comment:
These end up under the src/test directory because the annotation Processor needs them in the same source tree as the @TestableConnectors. Moving those instead to src/main is also a possibility.